<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edit_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> 전 송 </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_address">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>주 소: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용자 이름 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>등 록</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_text_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTnT_Chat">
    <property name="title">
     <string>TnT Chat</string>
    </property>
   </widget>
   <addaction name="menuTnT_Chat"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_send</sender>
   <signal>pressed()</signal>
   <receiver>edit_address</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
